src/custom_helpers.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+// All functions should be prefixed with "sc_"
+
+function sc_get_site_schema_markup(): ?string {
+    $seoService = new Sideways\Classic\Seo\SeoService();
+    return $seoService->get_site_schema_markup();
+}
+
+function sc_get_post_schema_markup(int $post_id = 0 ): ?string {
```

Comment:
Whats the purpose of the default param?

Reply:
it's an optional param for the seo plugin function which internally fetch the schema string using the postID param, otherwise first gets the postID using WP functions like get_the_id() or similar and then use that to get the schema